regression/contracts/loop-history-variables-03/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-all-contracts
+^PARSING ERROR$
+^EXIT=1$
+^SIGNAL=0$
+--
+--
+This test ensures that __CPROVER_loop_entry cannot be
+used for function contracts.
```

Comment:
But we are not checking that :) The C file is using `__CPROVER_old` in function contracts.